cmd/internal/collector/collector.go

```diff
@@ -189,3 +196,14 @@ func NewCCloudCollector() CCloudCollector {
 
 	return collector
 }
+
+func matchesRegex (candidate string, regex string) bool {
+	matched, err := regexp.MatchString(regex,candidate)
+	if err != nil {
+		panic(err)
```

Comment:
Not critical, but for consistency, maybe we should change this line by a `log.WithFields(log.Fields{"regex": regex, "candidate": candidate}).WithError(err).Fatalln("Can not execute regex on string string")

Reply:
I've actually made some improvements in regex matching so as to not need this anymore. If the regex can't be compiled, it will be done during config validation now :)